storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -54,8 +55,14 @@ using namespace xbcloud;
 
 const char *config_file = "my"; /* Default config file */
 
-enum { SWIFT, S3, GOOGLE };
-const char *storage_names[] = {"SWIFT", "S3", "GOOGLE", NullS};
+const char *azure_development_access_key =
+    "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/"
+    "KBHBeksoGMGw==";
+const char *azure_development_storage_account = "devstoreaccount1";
+const char *azure_development_container = "testcontainer";
```

Comment:
This should be made as parameters rather than hardcoded.

Reply:
azurite provide default keys.  If user is starting emulator with keys other default --azure-access-key works with --azure-development-storage.